web/packages/shared/components/UnifiedResources/shared/utils.ts

```diff
@@ -14,133 +14,116 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import { ResourceIconName } from 'design/ResourceIcon';
-
 import {
-  Icon,
   Application as ApplicationIcon,
   Database as DatabaseIcon,
   Kubernetes as KubernetesIcon,
   Server as ServerIcon,
   Desktop as DesktopIcon,
 } from 'design/Icon';
+import { ResourceIconName } from 'design/ResourceIcon';
 
 import { DbProtocol } from 'shared/services/databases';
 import { NodeSubKind } from 'shared/services';
 
 import {
-  UnifiedResourceKube,
-  UnifiedResourceNode,
+  UnifiedResourceItem,
   UnifiedResourceUi,
-  UnifiedResourceDatabase,
+  UnifiedResourceNode,
   UnifiedResourceApp,
-  UnifiedResourceUserGroup,
+  UnifiedResourceDatabase,
   UnifiedResourceDesktop,
-} from './types';
-
-export interface UnifiedResourceCard {
-  name: string;
-  description: {
-    primary?: string;
-    secondary?: string;
-  };
-  labels: {
-    name: string;
-    value: string;
-  }[];
-  primaryIconName: ResourceIconName;
-  SecondaryIcon: typeof Icon;
-  ActionButton: React.ReactElement;
-}
+  UnifiedResourceKube,
+  UnifiedResourceUserGroup,
+  SharedUnifiedResource,
+} from '../types';
 
-export function makeUnifiedResourceCardNode(
+export function makeUnifiedResourceItemNode(
```

Comment:
These functions no longer have to be exported.

Reply:
We still use them in `ResourceCard.story.tsx` and `ResourceListItem.story.tsx` in order to convert our `Node`, `App`, etc. fixtures to Unified Resource items.